src/main/java/com/shapesecurity/functional/data/NonEmptyImmutableList.java

```diff
@@ -63,6 +63,11 @@ public boolean equals(Object o) {
         // Manually expanded tail recursion
         ImmutableList<T> l = this;
         ImmutableList<T> r = (ImmutableList<T>) o;
+
+        if (l.length != r.length) {
+            return false;
+        }
```

Comment:
I am really confused how this change is showing up here, since it's already [in master](https://github.com/shapesecurity/shape-functional-java/blob/48b68fd0ce54d7e20142f2a367dbb3754a62a6ef/src/main/java/com/shapesecurity/functional/data/NonEmptyImmutableList.java#L67).

Reply:
I think GitHub shows you the diff from the parent commit, not the diff from the base branch of the PR.